### ISSUE TITLE
Uncaught ReferenceError: path is not defined

### ISSUE THREAD

**Issue by lee-dohm** [Osmose/advanced-open-file#138]
[Enter steps to reproduce:]

1. Launch Atom
2. Execute `advanced-open-file:toggle`

**Atom**: 1.16.0-dev-b68bd6683 x64
**Electron**: 1.3.13
**OS**: Mac OS X 10.12.3
**Thrown From**: [advanced-open-file](https://github.com/Osmose/advanced-open-file) package 0.16.5

### Stack Trace

Uncaught ReferenceError: path is not defined

```
At /Users/Lee/.atom/packages/advanced-open-file/lib/view.js:248

ReferenceError: path is not defined
    at AdvancedOpenFileView._appendToPathList (view.js:248:9)
    at AdvancedOpenFileView.setPathList (view.js:239:22)
    at AdvancedOpenFileView.pathEditor.onDidChange (view.js:44:18)
    at Function.module.exports.Emitter.simpleDispatch (/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/app.asar/node_modules/event-kit/lib/emitter.js:129:28)
    at /app.asar/src/text-editor.js:548:32
    at Function.module.exports.Emitter.simpleDispatch (/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/app.asar/node_modules/event-kit/lib/emitter.js:129:28)
    at DisplayLayer.emitDidChangeSyncEvent (/app.asar/node_modules/text-buffer/lib/display-layer.js:755:18)
    at /app.asar/node_modules/text-buffer/lib/text-buffer.js:681:29
    at Map.forEach (native)
    at TextBuffer.module.exports.TextBuffer.emitDidChangeEvent (/app.asar/node_modules/text-buffer/lib/text-buffer.js:680:41)
    at TextBuffer.module.exports.TextBuffer.applyChange (/app.asar/node_modules/text-buffer/lib/text-buffer.js:663:12)
    at TextBuffer.module.exports.TextBuffer.revertToCheckpoint (/app.asar/node_modules/text-buffer/lib/text-buffer.js:867:16)
    at TextBuffer.module.exports.TextBuffer.transact (/app.asar/node_modules/text-buffer/lib/text-buffer.js:821:14)
    at TextBuffer.module.exports.TextBuffer.setTextInRange (/app.asar/node_modules/text-buffer/lib/text-buffer.js:535:21)
    at TextBuffer.module.exports.TextBuffer.setText (/app.asar/node_modules/text-buffer/lib/text-buffer.js:475:19)
    at TextEditor.module.exports.TextEditor.setText (/app.asar/src/text-editor.js:1090:26)
    at AdvancedOpenFileView.setPath (view.js:203:25)
    at AdvancedOpenFileController.updatePath (controller.js:117:19)
    at AdvancedOpenFileController.attach (controller.js:328:14)
    at AdvancedOpenFileController.toggle (controller.js:236:18)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/app.asar/src/command-registry.js:259:29)
    at /app.asar/src/command-registry.js:3:59
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:599:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:390:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/app.asar/src/window-event-handler.js:106:36)
    at /app.asar/src/window-event-handler.js:3:59)
```

### Commands

```
     -0:13.3.0 advanced-open-file:toggle (input.hidden-input)
```

### Non-Core Packages

```
advanced-open-file 0.16.5 
atom-2048 3.1.0 
autocomplete-emojis 2.5.0 
autocomplete-html-entities 0.1.0 
badges 2.2.5 
block-travel 1.0.5 
bug-report 0.7.1 
convert-date 0.1.1 
docblockr 0.9.1 
double-tag 0.4.0 
file-type-icons 1.3.3 
fonts 2.0.0 
github 0.0.0 
github-atom-light-syntax 0.3.2 
hex 0.6.2 
highlight-selected 0.12.0 
indentation-indicator 1.0.0 
language-diff 0.7.0 
language-elixir 0.17.0 
language-generic-config 1.3.0 
language-pegjs 0.5.0 
language-rust 0.4.9 
language-slim 0.5.0 
less-than-slash 0.16.0 
markdown-helper 0.3.0 
markdown-mindmap 0.3.3 
minimap 4.26.8 
nebula-syntax 0.4.4 
nebula-ui 0.6.0 
package-sync 1.1.0 
pretty-json 1.6.1 
red-wavy-underline 0.3.2 
right-case 0.2.0 
set-syntax 0.3.2 
soft-wrap-indicator 1.0.1 
sort-lines 0.14.0 
tabs-to-spaces 1.0.3 
toggle-quotes 1.0.0 
unity-ui 2.1.8 
```

**Comment by Osmose** [comment]
Fixed by @lee-dohm and released as v0.16.6. Sorry for the delay on getting this out, and thanks for the help!

**Comment by lee-dohm** [comment]
You're welcome :grinning: I'm glad I could help :+1:
